pkg/sqlite/testdata/test_db_migrations/up_migration/194708150000_test_up_migration.down.sql

```diff
@@ -0,0 +1 @@
+/* Not required */
```

Comment:
For completions sake, maybe this should be an actual down migration? I see that the test isn't going to test this since it's probably hard to simulate an error here for right now, but I think we may want to revisit this later if we decided to add a `migrate` command.

Reply:
@kevinrizza there's no harm in adding a down migration. And in that note, I'm changing the folder name from `up_migration` to `migrations` to test both up and down migrations later.